src/fsharp/FSharp.Core/async.fs

```diff
@@ -1413,6 +1413,15 @@ namespace Microsoft.FSharp.Control
                 | _ ->
                     ctxt.econt edi)
 
+        static member Sleep (millisecondsDueTime: int32) : Async<unit> =
+            Async.Sleep (millisecondsDueTime |> int64)
+
+        static member Sleep (dueTime: TimeSpan) =
```

Comment:
This needs `[<Experimental(ExperimentalAttributeMessages.RequiresPreview)>]` added to it

Reply:
This turned out harder than expected.. I had to add ExperimentalAttributeMessages to the prim-types.fsi signature file because it wasn't accessible outside of prim-types.fs. Also it required begin/end syntax due to some weird bug with literals in signature files. But I think this is fixed now.